### ISSUE TITLE
Error in IE

### ISSUE THREAD

**Issue by mif007** [azazdeaz/react-gsap-enhancer#7]
Error from attachRefs.js in IE11.
Code that throws error:
```js
var item;
  if (itemMap.has(key)) {
    item = itemMap.get(key);
  } else {
    item = itemMap.set(key, {
      children: new Map()
    }).get(key);
  }
```
In IE11 code
```js
new Map().set(key,value)
```
returns undefined;

**Comment by azazdeaz** [comment]
Hi @mif007! Thanks, i'll check it as soon as i can!

**Comment by azazdeaz** [comment]
@mif007 
Fixed! (v 0.1.16 on npm) 

Thanks a lot for the helpful issue!
